src/assets/design-tokens.scss

```diff
@@ -1,15 +1,21 @@
 :root {
     // colors
     --color-primary: #415a77;
+    --color-primary-hover: #7d9abf;
     --color-secondary: #eceff1;
     --color-tertiary: #778da9;
     --color-headings: #1b263b;
-    --color-accent: #58af23;
-    --color-error: #d44;
+    --color-accent: #258839;
+    --color-accent-hover: #89cf5e;
+    --color-accent-hover-text: var(--color-headings);
+    --color-error: #c82d2d;
+    --color-error-hover: #f47171;
+    --color-error-hover-text: var(--color-headings);
     --color-warning: orange;
     --color-category-default: hsl(184, 54%, 70%);
     --color-category-website: hsl(285, 56%, 83%);
     --color-white: #fff;
+    --color-white-hover: #e4e8ee;
     --sidebar-color-1: var(--color-headings);
```

Comment:
niet in scope van deze pr, maar dit valt me nu op. misschien als restpunt opnemen om nog even de naamgeving en het gebruik van variabelen te reviewen. de naam van een css var zou niet moeten beschrijven waar hij gebruikt wordt maar welke rol hij heeft in het design

Reply:
#422 aangemaakt